### DISCUSSION TITLE
更新forge后无法使用

### DISCUSSION THREAD

**Discussion by EstraCcc** [IzzelAliz/Arclight#589] [Q&A]
1.18.2，强行更新forge后服务器无法成功开启，使用forge服务端，将arclight放入mods文件夹没有任何作用

**Comment by VeroFess** [comment]
你的目标版本是多少？

**Reply by EstraCcc** [reply]
1.18.2，forge 40.1.20 ，现降低mod版本后可以使用arclight自带40.1.0forge游玩，感谢回复。

**Comment by EstraCcc** [comment]
1.18.2Forge版本40.1.20

从 Windows 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>发送

发件人: ***@***.***>
发送时间: 2022年5月18日 11:40
收件人: ***@***.***>
抄送: ***@***.***>; ***@***.***>
主题: Re: [IzzelAliz/Arclight] 更新forge后无法使用 (Discussion #589)


你的目标版本是多少？

―
